knowage-vue/src/i18n/en_US.json

```diff
@@ -127,6 +128,44 @@
             "hint": "The metadata allows you to access and write metadata inside your users document. It's possible to change the type between the allowed ones and to name and describe it. Click on one existing item from the list on the side, or press the \"+\" button to add one.",
             "title": "Metadata Management"
         },
+        "profileAttributesManagement": {
+            "title": "Profile Attributes",
+            "name": "Name",
+            "description": "Description",
+            "validation": {
+              "extendedAlphanumeric": "Invalid format",
+              "maximumCharacters": "The maximum length allowed is {length}",
+              "required": "{field} is required"
+            },
+            "unsavedChangesHeader": "Unsaved changes",
+            "unsavedChangesMessage": "You have not saved your recent changes. Continue without saving ?",
```

Comment:
This label should be common in most list interfaces, move to a common.unsaved section

Reply:
Sorry, for to write you during your time off, but we already have the messages for confirm and unsaved changes in common.toast section and we have used it in roles-management components. Do you want as to remove it from that section and put it where you requested (common.unsaved and common.confirm)

Reply:
No sorry, you can use the one you already got here.
![image](https://user-images.githubusercontent.com/4438516/120612706-c1588400-c455-11eb-91d3-a5ce412ce542.png)
Just avoid proliferation when present.

Reply:
This should be resolved